easybuild/easyconfigs/q/QuantumESPRESSO/QuantumESPRESSO-6.0-intel-2016b.eb

```diff
@@ -34,6 +34,7 @@ source_urls = [
 
 patches = [
     'QuantumESPRESSO-5.4.0_yambo-fixes.patch',
+    'QuantumESPRESSO-6.0_yambo-fixes-foss-2016b.patch',
```

Comment:
Not sure if this patch would already resolve the same issues as the `5.4.0` patch, might be worth commenting out the first patch and seeing if this one works as a catch-all